src/styles/main.scss

```diff
@@ -1,3 +1,3 @@
-body {
```

Comment:
you haven't added all the style files, use `git add <file_name>`, then commit them and push, so I can review